### PR TITLE
Fix conceallevel setted globally instead of locally

### LINKED ISSUE #899: conceallevel is setted globally when editing a wiki file
# Version used and configuration
- Vimwiki : Stable version: 2.5
- VimWiki syntax : default
- nvim version : NVIM v0.5.0-416-g2b00d1d
- .vimrc config : 
```vim
NeoBundle 'vimwiki/vimwiki'
NeoBundle 'tbabej/taskwiki'
let g:vimwiki_global_ext = 0
let g:taskwiki_dont_preserve_folds = "yes"
let g:vimwiki_folding='custom'
```
# Bug description
When entering any wiki file, the conceallevel option is setted to 2 globally. It should be local.

### PR DESCRIPTION
**Pull request by PtitCaius**
# Proposed changes
The conceallevel option is now setted locally when entering a vimwiki buffer instead of globally

- [x] add local conceallevel option
- [x] run tests
- [x] edit changelog

Closes #899

### CODE REVIEW

File: doc/vimwiki.txt [hunk 1]
```diff
@@ -3521,7 +3521,8 @@ Contributors and their Github usernames in roughly chronological order:
     - Samir Benmendil (@Ram-Z)
     - Stefan Lehmann (@stevesteve)
     - @graywolf
-    - flex (@bratekarate)
+    - flex (@bratekarate)*
```
Comment (tinmarino) [inline doc/vimwiki.txt:3524]: Why a star here: is he a rock star ?

Comment (PtitCaius) [reply doc/vimwiki.txt]: Oops, just a typo I've made !

**Comment by tinmarino** [comment]
Perfect, I'll add some non-regression test.
Thank you @PtitCaius

**Comment by PtitCaius** [comment]
> Perfect, I'll add some non-regression test.
> Thank you @PtitCaius

You're welcome !

**Pull request merged**